### PR TITLE
Fix grammatical errors on en.json

### PR DESCRIPTION
**Pull request by JoaoJandre**
### Description

This PR aims to standardize and fix grammatical errors in the `locales/en.json` file to enhance the user experience. The following conventions were used:
Descriptions and tooltips end with `.`;
Ellipses use 3 dots;
Capitalization was standardized to start a phrase with a capitalized word and all the following words to be lowercase, unless when dealing with nouns.

Finally, a label was renamed to be more aligned with it's meaning.
### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [X] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
This was manually tested in a local lab, by trying to navigate most of the UI and see any errors or inconsistencies.

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
Rekicking for github action (not sure why it's not picking the workflow yaml from latest main)

**Comment by rohityadavcloud** [comment]
kicking manually 
@blueorangutan ui

**Review by utchoang — APPROVED** [review]

**Review by DaanHoogland — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

File: ui/public/locales/ar.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
 "label.action.delete.network": "Delete Network",
 "label.action.delete.network.processing": "Deleting Network....",
 "label.action.delete.nexusvswitch": "Delete Nexus 1000v",
-"label.action.delete.nic": "Remove NIC",
+"label.action.remove.nic": "Remove NIC",
```
Comment (DaanHoogland) [inline ui/public/locales/ar.json:96]: why is this change required? `delete` seems fine as a phrase.

Comment (JoaoJandre) [reply ui/public/locales/ar.json:96]: Everywhere else I saw this term as remove and not delete, and even the label text says remove. Futhermore, I think in ACS its more sensible to say remove instead of delete NIC, because in ACS the NICs cannot exist apart from a VM, it's not a separate entity, it can be added or removed from a VM, it cannot be created/destroyed in a vacuum. In the end, it's not that big of a deal, but it keeps the code cohesive.

Comment (DaanHoogland) [reply ui/public/locales/ar.json:96]: no, it isn´t a big deal, but a remove btween the deletes is as awkward as the delete with remove in the text. won´t :-1: , just want to say I don´t like this. it replaces noise with noise.
that said ; we should reorder these files alphabetically ;)

**Comment by nvazquez** [comment]
@blueorangutan ui

**Review by nvazquez — APPROVED** [review]
